src/main/java/by/exadel/internship/controller/InternshipController.java

```diff
@@ -71,4 +72,12 @@ public UserInternshipDTO restoreInternship(@PathVariable UUID internshipId) {
     public UserInternshipDTO saveInternship(@RequestBody UserInternshipDTO internshipDTO) {
         return internshipService.saveInternship(internshipDTO);
     }
+
+    @SuperAdminAccessControl
+    @PostMapping("/addUser")
+    @ApiOperation("Add User to Internship")
+    @ResponseStatus(HttpStatus.OK)
+    public boolean addUser(@RequestParam UUID userId, @RequestParam(name = "internshipId") UUID internshipId){
```

Comment:
Returning just boolean doesn't look like good solution

Reply:
change to void